tests/unit/ui/test_lifecycle.py

```
@@ -0,0 +1,74 @@
+
+def _build_app() -> SimpleNamespace:
+    return SimpleNamespace(
+        state_manager=SimpleNamespace(session=SimpleNamespace(user_config={})),
+        editor=object(),
+        chat_container=SimpleNamespace(write=lambda _renderable: None),
+        _request_worker=object(),
+        _slopgotchi_timer=None,
+        _update_resource_bar=lambda: None,
+        _update_slopgotchi=lambda: None,
+        set_focus=lambda _target: None,
+        run_worker=lambda _worker, exclusive=False: None,
```

Ruff (0.15.5)

[warning] Unused lambda argument: `exclusive`

(ARG005)